sdk/servicebus/azure-servicebus/stress/scripts/conftest.py

```diff
@@ -0,0 +1,46 @@
+#-------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#--------------------------------------------------------------------------
+import pytest
+import os
+import sys
+from azure.servicebus import ServiceBusClient
+from dotenv import load_dotenv
+
+LOGGING_ENABLE = False
```

Comment:
How verbose are your logs, btw? Wouldn't you want this on all the time?

Reply:
@benbp  - now that we're auto-logging all output into appinsights perhaps we need to think about some reasonable measure here? Any recommendations?

Reply:
@richardpark-msft I think we can keep an eye on it. @ckairen do you think it would be possible to graph and/or alert on high log counts per test or namespace? I think we only need to care if a test is uploading multiple GB/s per day, otherwise it's pretty cheap.

Reply:
https://github.com/Azure/azure-sdk-tools/issues/2776

Reply:
(storage costs aside, I'm mostly worried about someone uploading so much that they block ingestion)

Reply:
(Resolving this since the issue been filed)

Reply:
> (storage costs aside, I'm mostly worried about someone uploading so much that they block ingestion)

This would be extremely difficult :)